client/extensions/woocommerce/app/settings/email/mailchimp/getting-started.js

```diff
@@ -38,25 +41,43 @@ const GettingStarted = localize( ( { translate, onClick } ) => {
 					<h3 className="mailchimp__getting-started-list-header">
 						{ translate( 'Connect with your customers through MailChimp' ) }
 					</h3>
-					<ul className="mailchimp__getting-started-list">
+					<ul className="mailchimp__getting-started-list" >
 						{ list.map( ( item, key ) =>
 							<li key={ key }>
 								<Gridicon icon="checkmark" size={ 18 } />
 								{ item }
 							</li>
 						) }
 					</ul>
-					<Button className="mailchimp__getting-started-button" onClick={ onClick }>
-						{ translate( 'Get started with MailChimp' ) }
-					</Button>
+					{ ! redirectToSettings && (
+						<Button className="mailchimp__getting-started-button" onClick={ onClick }>
```

Comment:
Would it be cleaner to just have an onClick const that is assigned based upon `redirectToSettings`? Something like:

```
const mailChimpHref = redirectToSettings ? wizardLink : onClick;
...
<Button className="mailchimp__getting-started-button" onClick={ mailChimpHref }>
  { getStarted } // or not have getStarted as a const and just inline the translate string here
</Button>
```

Reply:
I don't think so. Once it is onClick and once it is href.